CPS-????/README.md

```diff
@@ -0,0 +1,93 @@
+---
+CPS: ?
+Title: Coin Selection Including Native Tokens
+Status: Open
+Category: Wallets
+Authors:
+  - Hinson Wong <wongkahinhinson@gmail.com>
+  - Tsz Wai Wu <wu.tszwai@outlook.com>
+Proposed Solutions:
+  - []
+Discussions:
+  - https://github.com/sidan-lab/CIPs/discussions
+  - https://discord.gg/uznpyACq
```

Comment:
```suggestion
Discussions:
  - https://github.com/cardano-foundation/CIPs/pull/611
```
Currently there are no discussions on this issue in your fork... although if there is a particular discussion about this specific issue then you're welcome to include it here.

Likewise, this is a link to the Gimbalabs Discord channel & too general to include here.  But we do have a precedent for linking from Discord threads into PR discussions, so you could note particular discussions here in the PR itself (and transcribe them please to GitHub when they are significant). 🤓

Reply:
Does it make more sense to link back to the issue #232 github page? So might continue discussion a year ago there

Reply:
Yes, I think that's a good idea... regardless of whether @nicolasayotte suggests including some of that discussion in the CIP, I think it should be added to the Discussions list (I'm adding it to my proposed changes above).